### PR TITLE
Added request queue page and connected to datastore to load pending requests.

### PR DESCRIPTION
**Pull request by imanherzi**
Created a page with all pending requests that links to a review page for each request. Deployed at /requests.html if you want to see it in detail.

### CODE REVIEW

File: bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import java.io.IOException;
+import java.lang.String;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/requests")
+public class RequestsServlet extends HttpServlet {
```
Comment (kzweller) [inline bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java:18]: Classes should have javadocs.

File: bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query<Request> q = ObjectifyService.ofy().load().type(Request.class);
```
Comment (kzweller) [inline bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java:21]: Spell out whatever "q" stands for.

File: bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    List<Request> allRequests = q.list();
+    List<Request> pendingRequests =
+        allRequests.stream().filter(req -> req.status == Status.PENDING).collect(toList());
```
Comment (kzweller) [inline bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java:24]: Should this filter by user ID? When is this servlet used? Things to mention in java doc.

File: bait-files/src/main/webapp/requests.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+const url =
+    'https://step-bait-project-2020.uc.r.appspot.com/review.html?requestId=';
+
+function getRequests() {
+  fetch('/requests').then(response => response.json()).then((requests) => {
+    const queue = document.getElementById('table-body');
```
Comment (kzweller) [inline bait-files/src/main/webapp/requests.js:6]: Consts here as well, as discussed, for HTML class and ID names

File: bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java (hunk 2 above)
Comment (tambchop) [reply bait-files/src/main/java/com/google/sps/servlets/RequestsServlet.java:21]: (I'm guessing it's "query")  =P

But I agree.  Full words are best!

File: bait-files/src/main/webapp/requests.js [hunk 5]
```diff
@@ -1,37 +1,41 @@
-const url =
+const TABLEBODY = 'table-body';
+const TABLEBODYCELL = 'table-body-cell';
+const TABLEBODYROW = 'table-body-row';
```
Comment (kzweller) [inline bait-files/src/main/webapp/requests.js:3]: Nit: Underscores between words

**Pull request merged**